package.json

```diff
@@ -80,6 +80,7 @@
     "proj4": "^2.5.0",
     "puppeteer": "^1.14.0",
     "replace-in-file": "^3.4.4",
+    "sinon": "^7.3.1",
```

Comment:
* You use Sinon to spy Console, What do you test exactly?

* I don't know if [startStaticFileServer](https://github.com/iTowns/itowns/blob/master/test/functional/bootstrap.js#L22) is fake server? if yes we could use [sinon fake server](https://sinonjs.org/releases/v7.3.1/fake-xhr-and-server/#fake-server).

* Sinon could be used to make mocks, too.

Reply:
Yes, `sinon` seems to be quite complete. I'm looking more into it then.

Reply:
I'm dropping `sinon` and letting the PR as it is without it.